packages/core/src/constants/crypto.ts

```diff
@@ -0,0 +1 @@
+export const CRYPTO_CONTEXT = "crypto";
```

Comment:
I noticed that we have few files like this, that just export on maybe 2 contexts for storage. Perhaps all of these can be combined into one file?